Dockerfile

```diff
@@ -11,7 +11,15 @@ FROM ubuntu:20.04 AS jbig2enc
 
 WORKDIR /usr/src/jbig2enc
 
-RUN apt-get update && apt-get install -y --no-install-recommends build-essential automake libtool libleptonica-dev zlib1g-dev git ca-certificates
+RUN apt-get update \
+  && apt-get install -y --no-install-recommends build-essential \
+    automake \
```

Comment:
That's the pattern of indentation I could observe in all other files, but in case you think there's a other convention, no problem of adopting

Reply:
The comment may be in the wrong place but it looks like automake is indented further than libtool. If this is the convention then that's fine by me. I personally would prefer all packages to have the same indentation.

Reply:
Some of this might stem from whitespace too.  `automake` is indented with 4 spaces, but most everything (though not all) in our Dockerfile is indented with tabs.

Tabs are what the `.editorconfig` would apply, so we should probably standardize to tabs indenting

Reply:
My eyeballs appreciate this too 😝